### PR TITLE
Calculator usability upgrade

### PR DESCRIPTION
**Pull request by beshaya**

### CODE REVIEW

File: src/components/calculator/SelectControl.tsx [hunk 1]
```diff
@@ -25,12 +25,36 @@ function showRiskMultiplier(multiplier: number): string {
   }
 }
 
+interface LabelFormat {
+  label: string
+  header?: string
+  subLabel?: string
+}
+
+export const labelMaker = function ({
```
Comment (beshaya) [inline src/components/calculator/SelectControl.tsx:34]: Maybe just make this a React Component? (see e.g. signature for GenericSelectControl).

Also, maybe call it something more descriptive, like LabelWithHeader?

Alternatively, this behavior could put put into GenericSelectControl.

File: src/pages/Calculator.tsx [hunk 2]
```diff
@@ -139,54 +142,73 @@ export const Calculator = (): React.ReactElement => {
     </span>
   )
 
+  function FirstTimeUserIntroduction() {
```
Comment (beshaya) [inline src/pages/Calculator.tsx:145]: This could be its own component + file.

File: src/styles/App.scss [hunk 3]
```diff
@@ -162,6 +156,16 @@ a {
       margin: 10px 0;
       background-color: $warning-fill;
     }
+    .calculator-params {
+      label {
+        @include media-breakpoint-up(lg) {
+          min-height: 3em;
```
Comment (beshaya) [inline src/styles/App.scss:162]: This doesn't work at all browser sizes:
image

Ok for now, I can see if I can figure out something with tables to make this more robust.

File: src/pages/Calculator.tsx (hunk 2 above)
Comment (blanchardjeremy) [reply src/pages/Calculator.tsx:145]: Done. ✅

File: src/styles/App.scss (hunk 3 above)
Comment (blanchardjeremy) [reply src/styles/App.scss:162]: Hm. Why tables? I thought the general thinking is table shouldn't be used for layout. That's what flexbox is for these days, I thought. Curious your thinking there.

File: src/components/calculator/SelectControl.tsx (hunk 1 above)
Comment (beshaya) [reply src/components/calculator/SelectControl.tsx]: (done)

File: src/styles/App.scss (hunk 3 above)
Comment (beshaya) [reply src/styles/App.scss:162]: I meant the <Row> and <Col> tags that come from bootstrap, not a literal table. But from the slack it sounds like the direction is a single column anyway, so this is moot.

Comment (blanchardjeremy) [reply src/styles/App.scss:162]: Yeah lets head toward 1 column. I'll do that in a separate PR though.

**Pull request merged**